File: misc/js/templates/cxErr.js
var template = template || {};

template.cxErr = function(msg) {
	return	"<div id=\"err-page\">\r\n"+
			"\t<h1>Sorry!</h1>\r\n"+
			"\t<p>Something went wrong.  We\'re getting the following error message:</p>\r\n"+
			"\t<p class=\"error\">"+(msg)+"</p>\r\n"+
			"\t<p>Please pay your bill through the waitstaff</p>\r\n"+
			"</div>";
};

File: server/Checkout Express/src/templates/CxErr.java
package templates;

public class CxErr {
	public static String run(Object msg) {
		return	"<div id=\"'err-page\"'>"+
			"\t<h1>Sorry!</h1>"+
			"\t<p>Something went wrong.  We\'re getting the following error message:</p>"+
			"\t<p class=\"'error\"'>"+(msg)+"</p>"+
			"\t<p>Please pay your bill through the waitstaff</p>"+
			"</div>";
	}
}